lighthouse-core/audits/load-fast-enough-for-pwa.js

```diff
@@ -63,6 +63,12 @@ class LoadFastEnough4Pwa extends Audit {
           return;
         }
 
+        // Disregard requests with an invalid start time, (H2 request start times are sometimes less
+        // than issue time and even negative which throws off timing)
+        if (record._startTime < record._issueTime) {
```

Comment:
checking this requires the checks from above to pass and seemed complicated enough to devote its own if check and comment